cvmfs/CMakeLists.txt

```diff
@@ -623,8 +634,8 @@ endif (BUILD_SERVER)
 
 if (BUILD_PRELOADER)
   install(
-    TARGETS      cvmfs_preload
-    RUNTIME
+    FILES      ${CMAKE_CURRENT_BINARY_DIR}/cvmfs_preload
```

Comment:
There is an integration test already that uses the installed version, by the way